brave/src/main/java/brave/Tracing.java

```diff
@@ -403,8 +406,19 @@ public Builder trackOrphans() {
     public Tracing build() {
       return new Default(this);
     }
+  }
 
-    Builder() {
+  static final class LogFinishedSpanHandler extends FinishedSpanHandler {
+    final Logger logger = Logger.getLogger(Tracer.class.getName());
+
+    @Override public boolean handle(TraceContext context, MutableSpan span) {
+      if (!logger.isLoggable(Level.INFO)) return false;
```

Comment:
@adriancole This logging created SO much noise in logs.
I suggest that it should be a lower level of logging (not INFO) or (and maybe in addition) let the logger be logger of `LogFinishedSpanHandler` and not the main `Tracer.class` logger so we can configure this logger specifically.
Currently the only way to reduce the noise is to change the `Tracer` logger to log only WARN and above.

Reply:
moving this question to here https://github.com/openzipkin/zipkin-support/issues/22 to avoid spamming here. you can also use https://gitter.im/openzipkin/zipkin next time.